torch_geometric/nn/models/lightgcn.py

```diff
@@ -110,7 +111,21 @@ def forward(self, edge_index: Adj,
                 :obj:`edge_index` will be used instead. (default: :obj:`None`)
         """
         if edge_label_index is None:
-            edge_label_index = edge_index
+            if isinstance(edge_index, SparseTensor):
+                edge_label_index = edge_index.to_torch_sparse_coo_tensor(
+                ).coalesce().indices()
+            elif isinstance(edge_index, torch.Tensor) and edge_index.is_sparse(
```

Comment:
I don‘t think we need to support this case right now. Any `tensor.is_sparse` logic should be dropped here.